resources/event-sources-controller-manager/templates/500-sources-controller.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2019 The Kyma Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: event-sources-controller-manager
```

Comment:
usually we use either {{ .Chart.Name }} or {{.Release.Name }} or some other name that is derived from whatever helm does for names of resources

Reply:
I wanted to avoid using variables for things are are invariable, but I'll do it to be compliant.

Reply:
Done